install_config/configuring_authentication.adoc

```diff
@@ -1391,3 +1391,55 @@ a secret with 32 or 64 bytes.
 <3> Encrypting secret, used to encrypt sessions. Must be 16, 24, or 32
 characters long, to select AES-128, AES-192, or AES-256.
 ====
+
+[[configuring-authentication-security]]
+== Security
```

Comment:
Better to change Line 1395 and 1396 to relate to user agent instead of authentication and security (I didn't notice this in last review)